src/Microsoft.AspNet.Mvc.Core/Rendering/IHtmlHelper.cs

```diff
@@ -82,6 +82,12 @@ public interface IHtmlHelper
         HtmlString AntiForgeryToken();
 
         /// <summary>
+        /// Returns anti-forgery cookie token and header.
```

Comment:
This should say sets the cookie and X-Frame-Options header on the response.

Add remarks for the use case of flush async